README.md

```diff
@@ -21,7 +21,7 @@ Mongolid ODM (Object Document Mapper) provides a beautiful, simple implementatio
 **Note:** If you are working with Laravel, take a look at [mongolid-laravel repository](https://github.com/leroy-merlin-br/mongolid-laravel).
 
 ## Requirements
-- PHP **7.1** or superior
```

Comment:
If have this version change, maybe should have a release with breaking change.
What about?

Reply:
This version had already changed at `composer.json`. I just adjust the README file.
![Captura de tela de 2022-09-19 18-48-37](https://user-images.githubusercontent.com/17056201/191126073-00abfe57-aff1-46d1-92d5-929e3cad65a2.png)